src/main/java/net/earthcomputer/clientcommands/command/AliasCommand.java

```diff
@@ -116,14 +118,14 @@ private static int addAlias(FabricClientCommandSource source, String key, String
         if (aliasMap.containsKey(key)) {
             throw ALIAS_EXISTS_EXCEPTION.create(key);
         }
-        if (ClientCommandInternals.getActiveDispatcher().getRoot().getChildren().stream().map(CommandNode::getName).anyMatch(literal -> literal.equals(key))) {
+        if (ClientCommandManager.getActiveDispatcher().getRoot().getChildren().stream().map(CommandNode::getName).anyMatch(literal -> literal.equals(key))) {
```

Comment:
How did this not break on Quilt?

Reply:
What do you mean?

Reply:
Using internals should have broken on Quilt because Quilt only implements the API part of Fabric. Anyway, doesn't matter

Reply:
That's what this code is for
```java
try {
    Class.forName("net.fabricmc.fabric.impl.command.client.ClientCommandInternals");
} catch (ClassNotFoundException e) {
    LOGGER.warn("Could not find ClientCommandInternals, /calias command not available");
    return;
}
```

Reply:
Looks like that code can be removed now.